README.md

```diff
@@ -1,15 +1,87 @@
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-svg/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-svg
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-svg/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-svg
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-svg.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-svg?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-svg.svg
+
+A component that renders the contents of an `SVG` file.
+
+## Installation
+
+```sh
+$ npm install @moxy/react-svg
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+In the course of making a project, there is often a need to inject `SVG` into the DOM. This package aims to simplify this matter, allowing for `SVG` inlining and `SVG` spriting.
 
 ## Usage
 
-This repo serves as the template for the creation of MOXY's base `React` components. To use this, just select `@moxystudio/react-lib-template` as the template when creating a new repo for your new package, and you're all set to start working.
+```js
+import React from 'react';
+import Svg from '@moxy/react-svg';
+
+import styles from './index.module.css';
+const arrowLeft = import(/* webpackChunkName: "svg-sprite" */ '../media/icons/arrow-left.inline.svg');
+
+const Home = () => {
+    const svgPromise = Promise.resolve({ default: arrowLeft });
+
+    return (
+        <div>
+            <h1>react-svg</h1>
+            <Svg svg={ svgPromise } className={ styles.svg } />
+        </div>
+    );
+};
+
+export default Home;
+```
+
+## API
+
+These are the props available in `@moxy/react-svg`.
+
+#### svg
+
+Type: `string` or `object` | Required: `true`
+
+The contents of the `SVG` that should be rendered. 
+In the case of the prop being an object, it must be the `Promise` that gets the contents of the `SVG`.
+
+#### classname
+
+Type: `string` | Required: `false`
+
+A className to apply to the component.
+
+#### forwardedRef
+
+Type: `object` | Required: `false`
+
+A ref that will be attached to the icon (`<svg>`) component that `@moxy/react-svg` ultimately returns.
+
+
+## Tests
 
-This template already includes a `src` folder, with 2 dummy files ready for you to start your work. `NewComponent` is a dummy component, available for demonstration purposes. Just rename `NewComponent.js` and change it according to your needs. An `index.js` for exporting is available as well. Do not forget to update the unit tests and try to reach as much coverage as possible.
+```sh
+$ npm test
+$ npm test -- --watch # during development
+```
 
-In order to help make proper use of this template, here's a quick checklist with some crucial stuff to have in mind:
+## License
```

Comment:
Before this line, add the `Demo` section that was added to the template (https://github.com/moxystudio/react-lib-template/pull/4).

After that, also add the `FAQ` that is going to be added to the template soon (https://github.com/moxystudio/react-lib-template/pull/5), might as well add it here now.